### PR TITLE
SetPreset PTZ command added

### PR DESCRIPTION
**Pull request by KevinBdn**

### CODE REVIEW

File: src/amcrest/ptz.py [hunk 1]
```diff
@@ -220,6 +220,17 @@ def go_to_preset(self, action=None, channel=0, preset_point_number=1):
         )
         return ret.content.decode('utf-8')
 
+    def set_preset(self, action=None, channel=0, preset_point_number=1):
```
Comment (pnbruckner) [inline src/amcrest/ptz.py]: I would add a `'start'` as the default value for `action`, like what I did when I added the `tour` method.

**Comment by pnbruckner** [comment]
Sorry, trying to figure out how to properly provide feedback. Haven't done that in github yet. 😄

File: src/amcrest/ptz.py (hunk 1 above)
Comment (dougsland) [reply src/amcrest/ptz.py]: > I would add a `'start'` as the default value for `action`, like what I did when
> I added the `tour` method.
I can't see the tour method in the ptz.py  :(  Could you please point?

Comment (dougsland) [reply src/amcrest/ptz.py]: have default as start looks good to me too. The code looks sane to me.

File: src/amcrest/audio.py [hunk 2]
```diff
@@ -107,23 +111,46 @@ def audio_stream_capture(self, httptype=None,
         """
         if httptype is None and channel is None:
             raise RuntimeError("Requires htttype and channel")
+        if not(type(delay) in [int,float] or (delay is None)):
```
[the remark this answers is not in the document]
Comment (pnbruckner) [reply src/amcrest/audio.py]: This is not very Pythonic.

File: src/amcrest/audio.py [hunk 3]
```diff
@@ -107,23 +111,46 @@ def audio_stream_capture(self, httptype=None,
...
+            raise RuntimeError("Delay must be float type or int type or None value")
+
 
         ret = self.command(
             'audio.cgi?action=getAudio&httptype={0}&channel={1}'.format(
                 httptype, channel), stream=True)
-
         if path_file:
             try:
+                if delay!=None:
```
[the remark this answers is not in the document]
Comment (pnbruckner) [reply src/amcrest/audio.py]: Also not Pythonic.

**Comment by pnbruckner** [comment]
This PR should be limited to adding the PTZ command. If you want to add another feature, please do so in a new PR.

**Comment by KevinBdn** [comment]
I know, sorry this is bad handling, I did not intend to submit this method as pull request.

**Pull request closed without merging**